### ISSUE TITLE
TypeError: Unknown block type: heb_print_multiple

### ISSUE THREAD

**Issue by zfi** [parallaxinc/solo#503]
When working with a Hackable Badge project, selecting the Communication->Badge Display-> Display Print Text block triggers the fault "TypeError: Unknown block type: heb_print_multiple". 

![2020-08-10 17_50_18-Settings](https://user-images.githubusercontent.com/8442260/90059156-923f0f00-dc97-11ea-816e-ac3acd2df8b8.png)

**Comment by zfi** [comment]
Tracking issue #504 here.

**Comment by zfi** [comment]
There are a number of places in the block definitions where a block definition is simply aliased to another block definition that we "assume" has already been defined. For example:
```
Blockly.Blocks.heb_print_multiple = Blockly.Blocks.oled_print_multiple;
```
When the source block has not yet been defined, the alias that is assigned to the new block definition is the value of the yet to defined block - UNDEFINED. This means that the order in which the block modules is imported is important. By swapping the order of heb.js and oled.js modules, this issue is resolved and all of the missing Badge Display blocks are available again.

**Comment by zfi** [comment]
Moved all of the block alias definitions into a new aliases module. This module is included after all of the other block definition modules have been imported. This will appear in v1.5.3.

**Comment by zfi** [comment]
Addressed in Solo 1.5.6
